runtime/core/decoder/torch_asr_decoder.cc

```diff
@@ -214,6 +214,39 @@ void TorchAsrDecoder::AttentionRescoring() {
       result_ = result;
     }
   }
+
+  if (opts_.bpe_decode && result_.size()) {
+    using std::string;
+    using std::vector;
+    vector<string> characters;
+
+    if (SplitUTF8String(result_, &characters)) {
+      string result;
+      const string delim = "\xe2\x96\x81";
```

Comment:
Make it a global constant variable in utils.h, kBlankSymbol?

Reply:
OK